presto-docs/src/main/sphinx/release/release-0.249.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.249
+=============
+
+**Highlights**
+==============
+* Add Support for Hive Connector to work with Hive 3 metastore for basic tables.
+* Update joda to 2.10.8 tzdata 2020d.
+* Support computing CRC32 checksum for data exchanges.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an error executing index joins when join spilling is enabled.
+* Enforce ``max_total_memory_per_node`` on user memory allocation.
+* Update joda to 2.10.8 tzdata 2020d. Make sure you deploy with a JVM that is using matching tzdata.
+* Support computing CRC32 checksum for data exchanges. Disabled by default and can be enabled by setting the `exchange.checksum-enabled=true` configuration property.
+
+Hive Changes
+____________
+* Add Support for Hive Connector to work with Hive 3 metastore for basic tables (ACID Tables and tables using constraints are not supported yet) (:pr:`15805`).
+* Fix a bug when S3 access is compromised across s3 enabled hive catalogs (:pr:`15800`).
```

Comment:
instead of "is compromised", I would say "is incorrectly cached" since it wasn't clear to me what compromised meant in this case.

also, put fixes first before other changes.